### PR TITLE
CCIT-343: Migrate lesson 1 for Platform 3

### PR DESCRIPTION
**Pull request by mvktsk**

### CODE REVIEW

**Comment by mvktsk** [comment]
Review task https://virtocommerce.atlassian.net/browse/VP-7737 has been created

**Review by BelyakovSergey — APPROVED** [review]

File: docs/lessons/index.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+﻿---
+title: Overview
+description: This lessons will show you basic of Virto Commerce and will also take you through some advance concepts related to VirtoCommerce application development
+layout: docs
+date: 2018-04-04
+priority: 1
+---
+## Welcome to the Virto Commerce lessons
+VirtoCommerce is an open source and .NET-based E-Commerce platform for B2B and B2C solutions.
+These lessons will teach you basic of VirtoCommerce and will also take you through some advance concepts related to VirtoCommerce application development.
```
Comment (aizhuli) [inline docs/lessons/index.md:10]: same as above.

**Review by aizhuli — APPROVED** [review]

**Review by OlegoO — COMMENTED** [review]
@mvktsk  Please move to either user-guide or getting-started folder. I think we need to create a new folder, if have at list 3 pages inside,

File: docs/lessons/index.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+﻿---
+title: Overview
+description: This lessons will show you basic of Virto Commerce and will also take you through some advance concepts related to VirtoCommerce application development
```
[the remark this answers is not in the document]
Comment (mvktsk) [reply docs/lessons/index.md:3]: Fixed

File: docs/lessons/index.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+layout: docs
+date: 2018-04-04
+priority: 1
+---
+## Welcome to the Virto Commerce lessons
+VirtoCommerce is an open source and .NET-based E-Commerce platform for B2B and B2C solutions.
```
[the remark this answers is not in the document]
Comment (mvktsk) [reply docs/lessons/index.md:9]: Fixed

File: docs/lessons/index.md (hunk 1 above)
Comment (mvktsk) [reply docs/lessons/index.md:10]: Fixed

File: docs/lessons/index.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+These lessons will teach you basic of VirtoCommerce and will also take you through some advance concepts related to VirtoCommerce application development.
+
+## Audience
+These lessons has been prepared for the beginners to help them understand basic VirtoCommerce.
```
[the remark this answers is not in the document]
Comment (mvktsk) [reply docs/lessons/index.md:13]: Not only platform. I think VirtoCommerce is suitable in this context.

**Comment by mvktsk** [comment]
> @mvktsk Please move to either user-guide or getting-started folder. I think we need to create a new folder, if have at list 3 pages inside,

@OlegoO  moved to getting-started

**Pull request merged**